MBBSEmu/Configuration.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.IO;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace MBBSEmu
+{
+    public static class AppSettings
```

Comment:
Let's not make this static, make it an instanced class that we can pass into classes using Dependency Injection

Reply:
Fixed.